lib/node_modules/@stdlib/stats/base/dists/logistic/ctor/benchmark/benchmark.js

```diff
@@ -30,95 +31,115 @@ var Logistic = require( './../lib' );
 
 // MAIN //
 
-bench( pkg+'::instantiation', function benchmark( bm ) {
-	var dist;
-	var mu;
-	var s;
-	var i;
-
-	bm.tic();
-	for ( i = 0; i < bm.iterations; i++ ) {
-		mu = ( randu() * 10.0 ) + EPS;
-		s = ( randu() * 10.0 ) + EPS;
-		dist = new Logistic( mu, s );
-		if ( !( dist instanceof Logistic ) ) {
-			bm.fail( 'should return a distribution instance' );
-		}
-	}
-	bm.toc();
-	if ( !( dist instanceof Logistic ) ) {
-		bm.fail( 'should return a distribution instance' );
-	}
-	bm.pass( 'benchmark finished' );
-	bm.end();
+bench( pkg+'::instantiation', function benchmark( b ) {
+    var mu;
+    var s;
+    var dist;
+    var len;
+    var i;
```

Comment:
```suggestion
    var dist;
    var len;
    var mu;
    var s;
    var i;
```

Variables are listed according to decreasing order of their lengths